app/selenium_ui/conftest.py

```diff
@@ -28,19 +28,19 @@
 
 def __get_current_results_dir():
     if 'TAURUS_ARTIFACTS_DIR' in os.environ:
-        return os.environ.get('TAURUS_ARTIFACTS_DIR')
+        return Path(os.environ.get('TAURUS_ARTIFACTS_DIR'))
     else:
         # TODO we have error here if 'results' dir does not exist
```

Comment:
This TODO is redundant in the current implementation.

Reply:
Done